cluster/images/conformance/go-runner/tar_test.go

```diff
@@ -73,6 +74,10 @@ func TestTar(t *testing.T) {
 			}
 
 			data, err := readAllTar(tc.outpath)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
why not use -
```
t.Errorf("Expected nil but got %v", err)
```
instead of Fatal.

Reply:
`t.Fatalf(...)` is the most appropriate here. `t.Errorf` will continue execution which doesn't make sense in this context. No reason to use `log.Fatal` in the test context; that is what `testing.T` is for.

Reply:
Yeah. agreed. I actually wanted to say to log and exit this test only - instead of log.Fatal. Thanks.

Reply:
Sorry about that.I changed the code.